save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphCollectionView.kt

```diff
@@ -140,23 +157,36 @@ val fossGraphCollectionView: FC<FossGraphCollectionViewProps> = FC { props ->
                                 }
                                 tab(selectedMenu.name, tabList, "nav nav-tabs mt-3") { value ->
                                     setSelectedMenu { VulnerabilityListTab.valueOf(value) }
-                                    setVulnerabilityFilters {
-                                        when (VulnerabilityListTab.valueOf(value)) {
-                                            VulnerabilityListTab.PUBLIC -> VulnerabilityFilter(
-                                                prefixName = "",
-                                                status = VulnerabilityStatus.APPROVED,
-                                            )
-
-                                            VulnerabilityListTab.ADMIN -> VulnerabilityFilter(
-                                                prefixName = "",
-                                                status = VulnerabilityStatus.CREATED,
-                                            )
-
-                                            VulnerabilityListTab.OWNER -> VulnerabilityFilter(
-                                                prefixName = "",
-                                                status = VulnerabilityStatus.CREATED,
-                                                isOwner = true,
-                                            )
+                                    when (VulnerabilityListTab.valueOf(value)) {
+                                        VulnerabilityListTab.PUBLIC -> {
+                                            setVulnerabilityFilters {
+                                                VulnerabilityFilter(
+                                                    prefixName = "",
```

Comment:
could it be the default value in `VulnerabilityFilter` itself?

Reply:
Changed it